### DISCUSSION TITLE
Does sfdx-git-delta works with sfv2.

### DISCUSSION THREAD

**Discussion by RohitGagan16** [scolladon/sfdx-git-delta#697] [Q&A]
Does sfdx-got-delta works with sfv2. As per the latest documention, sfdx-cli is deprecated and we should use sf based commands. so does sf(v2) supports the sgd-git-delta? if yes then whats the change to run the delta tool

**Comment by scolladon** [comment]
Hi @RohitGagan16 

Yes [it works with `sfv2`](https://github.com/scolladon/sfdx-git-delta/discussions/686#discussioncomment-6925644)! All the information you want are in this thread.

Happy incremental deployment 🚀
